docs/source/class/cf.Field.rst

```diff
@@ -442,13 +442,13 @@ CFA
    :toctree: ../method/
    :template: method.rst
 	      
+   ~cf.Field.file_locations
```

Comment:
Hi Sadie - anarchy rules! Sometimes it seems that alaphabetical makes sense, othertimes some logical grouping seems aprropriate. When (no if!) we refactor the whole documentation we'll look at this in the round.

Reply:
> anarchy rules

I have that on a T-shirt, with an exclamation at the front (not really, but it would work :smile: )!

Fair enough (when AI takes over it won't be able to automate such rules, so good thinking :grin: ). In that case, I'll generally not touch anything myself as to change the order (not that I had plans to, this would only be when otherwise touching the API reference method or class listings) unless it looks obviously out of alphabetical or (some) grouping order. Thanks for clarifying.